internal/web/app/index.js

```diff
@@ -156,7 +154,9 @@ document.addEventListener('DOMContentLoaded', () => {
           updateStatus(message);
           // Create a pre element to preserve whitespace and prevent wrapping
           const pre = document.createElement('pre');
-          pre.textContent = convert.toHtml(message);
+          // Strip ANSI color codes using regex
+          const strippedMessage = message.replace(/\x1b\[[0-9;]*m/g, '');
+          pre.textContent = strippedMessage;
```

Comment:
We were wrapping the message in `<pre>`. Replacing color codes with <span> makes no sense because they are rendered as literal strings. We simply remove the color codes here.